### PR TITLE
refactor(vue-app): simplify mount error log 

### PR DESCRIPTION
**Pull request by TheAlexLichter**
In case the mounting of the app fails, the error should be logged to the console in dev mode so it's easier to trace

## Types of changes
- [x] DX Improvements

## Checklist:
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly. (PR: #)
- [ ] I have added tests to cover my changes (if not applicable, please state why)
- [ ] All new and existing tests are passing.

### CODE REVIEW

**Review by pi0 — DISMISSED** [review]

**Comment by pi0** [comment]
@manniL For confirmation, dosen't it cause duplicate errors in console?

**Comment by manniL** [comment]
@pi0 	Stacktrace in FF w/o

image

With the error logged
image


It's easier to trace then :relaxed:

**Review by atinux — DISMISSED** [review]

**Review by clarkdo — COMMENTED** [review]
I'm a little confused, because we already log error in default errorHandler, so is it only for less stacktrace? If so, I think we'd better optimize original log format. 

The only difference is a error `wrapper`, may be we can change the wrapper to prepended message.

**Comment by pi0** [comment]
Maybe we can just pass the original error to errorHandler. `[error while mountin]` is not so useful for tracing.

File: packages/vue-app/template/client.js [hunk 1]
```diff
@@ -93,6 +93,7 @@ const errorHandler = Vue.config.errorHandler || console.error
 createApp()
   .then(mountApp)
   .catch((err) => {
```
Comment (clarkdo) [inline packages/vue-app/template/client.js]: @pi0 Then `.catch(errorHandler)` is enough

Comment (pi0) [reply packages/vue-app/template/client.js]: Yes. Agree. Cleaner code :)

File: packages/vue-app/template/client.js [hunk 2]
```diff
@@ -93,6 +93,7 @@ const errorHandler = Vue.config.errorHandler || console.error
...
+    <% if (isDev) { %>console.error(err)<% } %>
```
Comment (clarkdo) [inline packages/vue-app/template/client.js]:
```js
createApp().then(mountApp).catch(errorHandler)
```

File: packages/vue-app/template/client.js (hunk 1 above)
Comment (clarkdo) [reply packages/vue-app/template/client.js]:
```js
createApp().then(mountApp).catch(errorHandler)
```

Comment (manniL) [reply packages/vue-app/template/client.js]: Sure, makes sense :)

**Pull request merged**